samples/benchmarks/coremark/boards/nrf54h20dk_nrf54h20_cpuapp.conf

```
@@ -8,7 +8,7 @@ CONFIG_COREMARK_ITERATIONS=10000
 # Kconfigs required for the STM standalone logging, imported from the nordic-log-stm snippet.
 CONFIG_TEST_LOGGING_DEFAULTS=n
 CONFIG_LOG_FRONTEND=y
-CONFIG_LOG_FRONTEND_ONLY=y
+CONFIG_LOG_FRONTEND_ONLY=no
 CONFIG_LOG_FRONTEND_STMESP=y
 CONFIG_LOG_FRONTEND_STMESP_FSC=y
 
```

- The change from `CONFIG_LOG_FRONTEND_ONLY=y` to `CONFIG_LOG_FRONTEND_ONLY=no` may lead to unintended consequences if the frontend relies on this setting being enabled. Please reconsider this modification. 

```suggestion
CONFIG_LOG_FRONTEND_ONLY=y
```